UIInfoSuite2/UIElements/ShowQueenOfSauceIcon.cs

```diff
@@ -158,9 +158,7 @@ private void LoadRecipes()
                     string[] values = next.Value.Split('/');
                     if (values.Length > 1)
                     {
-                        _recipesByDescription[values[1]] = _helper.Content.CurrentLocaleConstant == LocalizedContentManager.LanguageCode.en || values.Length < 3
-                            ? values[0]
-                            : values[2];
```

Comment:
I think this condition has an important part, and was introduced as part of this PR:  https://github.com/Annosz/UIInfoSuite2/pull/155

values[0] is always English, but with different game languages we want to show the local name for the recipe: https://github.com/Annosz/UIInfoSuite2/issues/115

Reply:
So yeah, if the bug is caused by _todaysRecipe being English <-> other language string, then we need to have some more complex solution, because switching to English will cause the previous problem to reappear.

Reply:
I'm aware of it, which is why I'm using `_todaysRecipe.name` (which is in English) for the function calls, but `_todaysRecipe.DisplayName` (which is localized) for the display in `OnRenderedHud()`.

Reply:
Thanks, and sorry, so far I was only doing the review by reading, I will run the code as soon as I have the time :)